shell/platform/darwin/macos/framework/Source/AccessibilityBridgeMac.mm

```diff
@@ -187,9 +187,7 @@
       if (ax_node.data().HasState(ax::mojom::State::kEditable)) {
```

Comment:
@chunhtai Do you know a widget that exercises this scenario? This scenario is for nodes with the `kEditable` state that don't have the `kTextField` role. However, the [logic to create `AXNode`s](https://github.com/flutter/engine/blob/e143b309cc506a94fb3d4ae75c89794eda4a4885/shell/platform/common/accessibility_bridge.cc#L313-L382) only [gives the `kEditable` state](https://github.com/flutter/engine/blob/e143b309cc506a94fb3d4ae75c89794eda4a4885/shell/platform/common/accessibility_bridge.cc#L368-L371) to semantic nodes with the `FlutterSemanticsFlag::kFlutterSemanticsFlagIsTextField` flag, and [any such semantics node would have the `kTextField` role](https://github.com/flutter/engine/blob/e143b309cc506a94fb3d4ae75c89794eda4a4885/shell/platform/common/accessibility_bridge.cc#L320-L324). In other words, it seems this code is unreachable. Please let me know if I misunderstood something.

Reply:
Is there documentation in the mojo enum for this role? It looks like other things I can think of that are editable all have their own values (`kCheckbox`, `kSlider`, etc.).

The `ax::mojo::State` enum is defined as strictly powers of two? i.e. there's no state `kFoo` that happens to be `kBar | kEditable` I assume?

Reply:
Looks like these are defined as `enum class` so the answer appears to be that at least today, this is unused code.

Reply:
Oh just to make sure it's clear: semantic nodes have both `state` bit flags and a `role` enum. These are separate :)

Reply:
This is copied code from chromium, I don't think flutter have such widget that would run this code. We may in the future though, and we would need to update the common a11y_bridge to support that, but I am ok if we delete this code path.

Reply:
Gotcha. I'll leave it in for now given we may have such a widget in the future.